### ISSUE TITLE
Folded nodes count.

### ISSUE THREAD

**Issue by amariusz** [freeplane/freeplane#1320]
When part of the map is folded there's a dot (or circle depending on `Folding mark width`) on the right side of the folded node. I think it would be useful to have an option in preferences to turn on displaying the number of folded nodes next to this dot or within that circle. 

Right now this could be simulated with an attribute and simple formula like `=node.find{true}.size()-1`, but adding it to every node is not very convenient.

An example of this feature in other mind mapping software: https://youtu.be/me11dazcnZU?t=159

There's another issue regarding node count, but not as a live preview.
https://github.com/freeplane/freeplane/issues/1164

**Comment by theelectricsoul** [comment]
You will have much more luck if you explain why this feature is useful.

**Comment by amariusz** [comment]
When you work with big folded mindmap, especially one you don't know well or revisit only from time to time, it provides a hint if exploring a folded branch will take you something like 10s or rather 10min. 

Additionally it may serve as indicator during mind map design telling you if subtrees are balanced (similar) in size or not.

**Comment by euu2021** [comment]
yes, this would be useful, indeed. In the youtube video linked by OP it's possible to notice that the number of child nodes allows the user to "feel" the map better. As the presenter navigates through the map, even though I've never seem that map, I can get an idea of the actual size of the branches. For example, just by looking at the folded nodes, it's possible to get an idea of which of the branches are bigger:

![image](https://github.com/freeplane/freeplane/assets/77707706/3ce53376-28b6-4a2d-a58d-3408a810f317)

I think it has the potential to improve the UX, the flow in the usage, because it reduces the need of constant fold/unfold in order to get a general idea of the map

**Comment by aaa1386** [comment]
It is a very good idea

**Comment by euu2021** [comment]
see https://github.com/freeplane/freeplane/discussions/2956
